Source/WebCore/platform/graphics/ca/GraphicsLayerCA.cpp

```diff
@@ -2375,10 +2378,9 @@ void GraphicsLayerCA::updateSublayerList(bool maxLayerDepthReached)
     };
 
     auto appendLayersFromChildren = [&](PlatformCALayerList& list) {
-        for (const auto& layer : children()) {
-            const auto& currentChild = downcast<GraphicsLayerCA>(layer.get());
-            PlatformCALayer* childLayer = currentChild.layerForSuperlayer();
-            list.append(childLayer);
+        for (const auto& childLayer : children()) {
+            Ref currentChild = downcast<GraphicsLayerCA>(childLayer.get());
+            list.append(currentChild->layerForSuperlayer());
         }
```

Comment:
Would something like this work?
```suggestion
       for (Ref child : children())
           list.append(downcast<GraphicsLayerCA>(child)->layerForSuperlayer());
```

Reply:
I cannot easily verify locally but I can try on EWS.

Reply:
Looks like this worked fine.